spec/unit/blockscore/candidate_spec.rb

```diff
@@ -1,46 +1,192 @@
+require 'faker'
+
 module BlockScore
   RSpec.describe Candidate do
-    let(:api_stub) { @api_stub }
+    describe '.new' do
+      subject(:candidate) do
+        BlockScore::Candidate.new(
+          ssn: '0000',
+          date_of_birth: Faker::Date.backward(365 * 100).to_s,
+          name_first: Faker::Name.first_name,
+          name_last: Faker::Name.last_name,
+          address_street1: Faker::Address.street_address,
+          address_city: Faker::Address.city,
+          address_country_code: Faker::Address.country_code
+        )
+      end
+
+      it { is_expected.not_to be_persisted }
+      it { expect(candidate.class).to be BlockScore::Candidate }
+    end
+
+    describe '.create' do
+      subject(:candidate) do
+        BlockScore::Candidate.create(
+          ssn: '0000',
+          date_of_birth: Faker::Date.backward(365 * 100).to_s,
+          name_first: Faker::Name.first_name,
+          name_last: Faker::Name.last_name,
+          address_street1: Faker::Address.street_address,
+          address_city: Faker::Address.city,
+          address_country_code: Faker::Address.country_code
+        )
+      end
+
+      it { is_expected.to be_persisted }
+      it { expect(candidate.class).to be BlockScore::Candidate }
+    end
+
+    pending '.find' do
+      context 'valid candidate id' do
+        let(:candidate_id)  { create(:candidate).id }
+        subject(:candidate) { BlockScore::Candidate.find(candidate_id) }
+
+        it { is_expected.to be_persisted }
+        it { expect(candidate.name_first).not_to be_empty }
+      end
+
+      context 'invalid candidate id' do
+        let(:candidate_id) { '6c89646eea50fcaa42ca1fe1667a470b' }
+
+        it { expect { BlockScore::Candidate.find(candidate_id) }.to raise_error BlockScore::RecordNotFound }
+      end
+    end
+
+    describe '.retrieve' do
+      let(:candidate_id)  { create(:candidate).id }
+      subject(:candidate) { BlockScore::Candidate.retrieve(candidate_id) }
+
+      it { is_expected.to be_persisted }
+      it { expect(candidate.name_first).not_to be_empty }
+      it { expect(candidate.class).to be BlockScore::Candidate }
```

Comment:
since we have `rspec-its` as a dependency you can always write this as 

```ruby
its(:class) { should be(BlockScore::Candidate) }
```

I'm a fan of this when it is possible